docs/index.md

```diff
@@ -17,11 +17,11 @@ sure which to use, the client credentials grant is recommended, as it was design
 1. Create a new client using the `openid-connect` protocol. This client can be created in the `master` realm if you would
 like to manage your entire Keycloak instance, or in any other realm if you only want to manage that realm.
 1. Update the client you just created:
-    1. Set "Access Type" to "confidential".
-    1. Set "Standard Flow Enabled" to "OFF".
-    1. Set "Direct Access Grants Enabled" to "OFF"
-    1. Set "Service Accounts Enabled" to "ON".
-1. Grant required roles for managing Keycloak via the "Service Account Roles" tab in the client you created in step 1.
+    1. Set `Access Type` to `confidential`.
+    1. Set `Standard Flow Enabled` to `OFF`.
+    1. Set `Direct Access Grants Enabled` to `OFF`
+    1. Set `Service Accounts Enabled` to `ON`.
+1. Grant required roles for managing Keycloak via the `Service Account Roles` tab in the client you created in step 1. To do this, select `realm-management` in the `Client Roles` drop down menu and add required roles to the client or just assign `realm-admin` for full access.
```

Comment:
This is actually covered in the section below called "Assigning Roles".  Could you take a look at that and see if it makes sense to tweak that a little bit?  I'm also fine with adding a link here that points to that other section, but I'd prefer not to duplicate its content.

Reply:
Oh you are right! I just didn't read that bit, my fault.

> I'm also fine with adding a link here that points to that other section

Better idea, I'll update the PR.

Reply:
Done